content/doc/addons/postgresql/_index.md

```diff
@@ -18,42 +18,42 @@ aliases:
 
 {{< readfile file="db-backup.md" >}}
 
 ## Migrating from an old database
 
-Some applications require a populated database to run properly.
-If you want to import your **SQL** dump, you can use several methods:
-
-1. [Our WebGUI (Adminer)](https://dbms-adminer.clever-cloud.com/).
-2. Command line tool for PostgreSQL administration like `psql`.
-3. Any PostgreSQL client such as [pgAdmin](https://www.pgadmin.org/).
+Some applications require a non-empty database to run properly. If you want to import your **SQL** dump, you can use several methods:
+1. [WebGUI (Adminer)](https://dbms-adminer.clever-cloud.com/)
+2. Command line tool for PostgreSQL administration like `psql`
+3. Any PostgreSQL client such as [pgAdmin](https://www.pgadmin.org/)
 
 ## Direct access
 
-All our dedicated PostgreSQL databases are served via a proxy. To reduce the latency you can bypass this proxy by generating direct hostname and port for the addon. You can do it by clicking the "Generate direct hostname and port" on the addon dashboard.
+A proxy serves all dedicated PostgreSQL databases. In some cases, this can add some latency between applications and their database. If this is an issue, you can generate a direct hostname and port for the add-on to bypass the proxy, using the "Generate direct hostname and port" button in the add-on dashboard.
+
+Generating direct access adds new variables to the add-on's environment, allowing connections without going through the proxy.
 
-This action will add new environment variables to reach the addon without any proxy.
+{{< callout type="info" >}} 
+Direct access doesn't persist after migrating a database. Manually re-generate direct host name and port after a migration to allow applications linked through direct access to connect to the database. 
 
-Keep in mind that usage of direct access is a trade-off: when you migrate your addon, you will need to to generate hostname and port again so your application must update these environment while proxy usage changes nothing.
+Such a limitation doesn't exist when using proxy access.
+{{< /callout >}}
 
 ## Encryption at rest
 
-Encryption at rest is available on PostgreSQL. You can have more information on the [dedicated page]({{< ref "doc/administrate/encryption-at-rest.md" >}})
+Encryption at rest is available on PostgreSQL. You can find more information on the [dedicated page]({{< ref "doc/administrate/encryption-at-rest.md" >}})
 
 ## Note on shared databases
 
-Free databases are instantiated from a shared cluster. The way postgresql works on shared clusters is that a "pg_database" table references the name, and only the name, of other databases in that same cluster. The data, on the other hand, is not accessible.
+Free databases are instantiated from a shared cluster. The way postgresql works on shared clusters is that a `pg_database` table references the name, and only the name, of other databases in that same cluster. The data, on the other hand, is not accessible.
```

Comment:
```suggestion
If you try to list the databases on the shared cluster, you will see the names of the databases of all other users of the cluster. This is normal behaviour. Rest assured, permissions are set properly, meaning no one but your user can read or write data to your database.
```

@aurrelhebert, is this still the case? I have a feeling it was patched recently :thinking:

Reply:
I let you decide on this and tell me if it's ok for commit/merge or not

Reply:
Not released yet ;)

Reply:
Okay, then @davlgd you can apply the suggestion and then I'll approve :)

Reply:
Okay, then @davlgd you can apply the suggestion and then I'll approve :)